compiler/src/dotty/tools/dotc/transform/init/Objects.scala

```diff
@@ -66,7 +66,9 @@ import dotty.tools.dotc.core.Flags.AbstractOrTrait
  *     whole-program analysis. However, the check is not modular in terms of project boundaries.
  *
  */
-class Objects:
+class Objects(using Context @constructorOnly):
+  val immutableHashSetBuider: Symbol = requiredClass("scala.collection.immutable.HashSetBuilder")
+  val HashSetBuilder_rootNode: Symbol = immutableHashSetBuider.requiredValue("rootNode")
```

Comment:
```suggestion
  // TODO: this should really be an annotation on the rhs of the field initializer rather than the field itself.
  val HashSetBuilder_rootNode: Symbol = immutableHashSetBuider.requiredValue("rootNode")
```